metricbeat/module/docker/container/data.go

```diff
@@ -17,24 +16,45 @@ func eventsMapping(containersList []dc.APIContainers) []common.MapStr {
 	}
 	return myEvents
 }
-func eventMapping(mycontainer *dc.APIContainers) common.MapStr {
+
+func eventMapping(cont *dc.APIContainers) common.MapStr {
 
 	event := common.MapStr{
-		"@timestamp": time.Now(),
+		"timestamp": time.Now(),
```

Comment:
@douaejeouit I saw this part, but we are talking here about a milli second difference in time. I don't think it is worth to do this and have the logic because of this different from all other modules / metricsets where the timestamp represents when the event fetching started. The nice part of this is that `timestamp + rrt` is when the even creation ended. With the above implementation it is not 100% clear to me what the timestamp will exactly represent.

@douaejeouit  Perhaps you can share some insights on what the timestamp exactly is.